content/reference-transaction-format.md

```diff
@@ -157,6 +178,16 @@ After a transaction has been submitted, if it gets accepted into a validated led
 }
 ```
 
+
+### Multi-Signing ###
+
+Multi-signing in Ripple is the act of [authorizing transactions](#authorizing-transactions) for the Ripple Consensus Ledger by using a combination of multiple secret keys. Multi-signing is due to be enabled by an [Amendment](concept-amendments.html) to the Ripple Consensus Protocol. You can use multi-signing in addition to, or instead of, a master key, a [regular key](#setregularkey), or both.
```

Comment:
The last part of the last sentence is slightly unclear to me. You can use multi-signing and one or both of the other methods, or you can use multi-signing in addition to one or both of the other methods?